packages/suite/src/components/suite/Preloader/Preloader.tsx

```diff
@@ -61,6 +62,8 @@ export const Preloader = ({ children }: PropsWithChildren) => {
     const isFirmwareAuthenticityCheckDismissed = useSelector(
         selectIsFirmwareAuthenticityCheckDismissed,
     );
+    // report firmware authenticity failures even when the UI is disabled
```

Comment:
We discussed on slack that we will leave it, because the suggested name also describes _expected usage_ instead of describing what it does (preferred).